mojave/machinery/machinery.dm

```diff
@@ -87,20 +87,19 @@
 	broadcasting = FALSE  // Whether the radio will transmit dialogue it hears nearby.
 	freerange = TRUE  // If true, the radio has access to the full spectrum.
 	freqlock = TRUE  // Frequency lock to stop the user from untuning specialist radios.
+	anonymize = TRUE
 	radio_broadcast = RADIOSTATIC_LIGHT
 
 /obj/item/radio/intercom/ms13/Initialize(mapload)
 	. = ..()
-	wires = new /datum/wires/radio(src)
-	if(prison_radio)
-		wires.cut(WIRE_TX) // OH GOD WHY
-	secure_radio_connections = new
-	for(var/ch_name in channels)
-		secure_radio_connections[ch_name] = add_radio(src, GLOB.radiochannels[ch_name])
```

Comment:
As Original explained in Discord, this is for the functionality of intercoms to not allow people to change the intercom.

Reply:
no this dosent do anything because its listed in the base radios init, shrimply its just copy and pasted fartcode. The freqlock variable is the thing that stops it changing it

Reply:
and yes I tested that